### ISSUE TITLE
Changes to `gh auth login -s 'user:email' -w`

### ISSUE THREAD

**Issue by Disco-Chef** [lewagon/setup#367]
Hello team!

It looks like with `gh auth login -s 'user:email' -w`, students can also set up an SSH key and give it to Github in one go.

This means that the following section (https://github.com/lewagon/setup/blob/master/windows.md#ssh-key) is potentially no longer necessary?

Personally I would do this, because I am set up on multiple machines and I like to give useful names to my keys on Github:
- `mkdir -p ~/.ssh && ssh-keygen -t ed25519 -o -a 100 -f ~/.ssh/id_ed25519 -C "STUDENT_EMAIL@HERE.com"`
- `cat .ssh/id_ed25519.pub`
- Give the public key to github (https://github.com/settings/keys)
- `ssh -T git@github.com`
- `gh auth login -s 'user:email' -w`, choose SSH and skip giving public key to Github
- `gh auth status` for final check

However, all of this could be done with just `gh auth login -s 'user:email' -w` but they will get no say in the title of the key (as has been the case for a while now).

I would want us to promote them to name their keys, seeing as it will help them distinguish their private machine from their company one (in the future, that is, once they finish the bootcamp). But for the case of convenience I guess it's low priority.

TL;DR: We can merge these sections into one:
- https://github.com/lewagon/setup/blob/master/windows.md#github-cli
- https://github.com/lewagon/setup/blob/master/windows.md#ssh-key

and have them just use `gh auth login -s 'user:email`, as long as we point out the right options to pick from the prompts (and naturally the check afterwards too).

![image](https://user-images.githubusercontent.com/49205098/160341145-d5c417f9-68f5-4eae-affc-7e0175c1d0cd.png)

(in this image, I chose to skip since I did it in my own way, but you get the idea 😄 )

🙇 🖖

**Comment by Disco-Chef** [comment]
Duplicate Issue, had not noticed. Closing.
